### DISCUSSION TITLE
OrderBy into related table

### DISCUSSION THREAD

**Discussion by donPuerto** [supabase/supabase#2966] [Questions] [answered]
Hi I tried to sort created_by column into a relation table but got no luck results.
...
scenario
i have 2 tables User and Phones, One to many relationship
...
===User===
id
first_name
last_name
created_at
updated_at

===Phones===
id
user_id
phone
created_at
updated_at

Now, I want to sort the Phone tables at created_at column

```js
const { data, error, status } = await supabase
          .from('users')
          .select(`
            *,
            phones(*)
          `)
          .order('created_at', {
            foreignTable: 'users.phones',
            ascending: true}
          )
          .eq('id', id)
          .single()
```

But above code its not working

**Comment by soedirgo** [comment, marked as answer]
Can you try setting `foreignTable` to `'phones'`?

**Reply by donPuerto** [reply]
Yes its working mate.
